src/routes/comment.route.ts

```diff
@@ -13,8 +13,8 @@ class CommentRoute implements Routes {
   }
 
   private initializeRoutes() {
-    // test
     this.router.get(`${this.path}`, AuthMiddleware, this.commentController.getComments);
+    this.router.post(`/posts/:post_id${this.path}`, AuthMiddleware, this.commentController.postComment);
```

Comment:
Nit: Are we not grouping routes based on their name? I thought we were, so I would assume this would actually be on the post routes so it would be like (`{this.path}:post_id/comments`). I could be mistaken but we should confirm the format with the others. Bring it up next time.